conf/gateway-testing.ini

```diff
@@ -102,23 +102,12 @@ poll.interval = 1 s
 ;Periodic interval for sending of statistics
 statistics.interval = 10 s
 
-[hotplug]
-pipe.path = ${application.configDir}../gateway.hotplug
-impl = udev
+[hotplug]                                                                       
+pipe.path = /var/run/beeeon-gateway.hotplug                                     
+impl = pipeHotplug
 
```

Comment:
Tyto změny sem určitě také nepatří. Při instalaci do LEDE vytvoř konfigurační soubor v /etc/beeeon/gateway/config.d/lede.ini a do něj umísti tyto 3 řádky. Soubor se načte automaticky.